conda_forge_tick/update_prs.py

```diff
@@ -188,6 +199,13 @@ def main(args: "CLIArgs") -> None:
         # This function needs to run last since it edits the actual pr json!
         gx = close_dirty_prs(gx, args.dry_run)
 
+    # stop profiler
+    prof.disable()
+
+    # output to data
+    os.makedirs("profiler", exist_ok=True)
+    prof.dump_stats(f"profiler/update_prs_{current_time}.")
```

Comment:
can this go in a subfolder (and put the other in a subfolder)?

Reply:
sure, I will modify that. Thanks for the advice 😄

Reply:
done 😄